testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1346,3 +1369,95 @@ def server_secret
   log "Reactivation Key: #{react_key}"
   step %(I enter "#{react_key}" as "reactivationKey")
 end
+
+When(/^I prepare Cobbler for the buildiso command$/) do
+  tmp_dir = "/var/cache/cobbler/buildiso"
+  $server.run("mkdir -p #{tmp_dir}")
+  # we need bootloaders for the buildiso command
+  _out, code = $server.run("cobbler mkloaders")
+  raise 'error in cobbler mkloaders, check the Cobbler logs' if code.nonzero?
+end
+
+When(/^I run Cobbler buildiso for distro "([^"]*)" and all profiles$/) do |distro|
+  tmp_dir = "/var/cache/cobbler/buildiso"
+  iso_dir = "/var/cache/cobbler"
+  _out, code = $server.run("cobbler buildiso --tempdir=#{tmp_dir} --iso #{iso_dir}/profile_all.iso --distro=#{distro}")
+  raise 'error in cobbler buildiso, check the Cobbler logs' if code.nonzero?
+  profiles = ["orchid", "flame", "pearl"]
+  isolinux_profiles = []
+  cobbler_profiles = []
+  profiles.each do |profile|
+    # get all profiles from Cobbler
+    result_cobbler, code = $server.run("cobbler profile list | grep -o #{profile}")
+    cobbler_profiles.push(result_cobbler) if code.zero?
+    # get all profiles from isolinux.cfg
+    result_isolinux, code = $server.run("cat #{tmp_dir}/isolinux/isolinux.cfg | grep -o #{profile} | cut -c -6 | head -n 1")
+    unless result_isolinux.empty?
+      isolinux_profiles.push(result_isolinux)
+    end
+  end
+  raise 'error during comparison of Cobbler profiles' unless cobbler_profiles == isolinux_profiles
+end
+
+When(/^I run Cobbler buildiso for distro "([^"]*)" and profile "([^"]*)"$/) do |distro, profile|
+  tmp_dir = "/var/cache/cobbler/buildiso"
+  iso_dir = "/var/cache/cobbler"
+  _out, code = $server.run("cobbler buildiso --tempdir=#{tmp_dir} --iso #{iso_dir}/#{profile}.iso --distro=#{distro} --profile=#{profile}")
+  raise 'error in cobbler buildiso, check the Cobbler logs' if code.nonzero?
+end
+
+When(/^I run Cobbler buildiso for distro "([^"]*)" and profile "([^"]*)" without dns entries$/) do |distro, profile|
+  tmp_dir = "/var/cache/cobbler/buildiso"
+  iso_dir = "/var/cache/cobbler"
+  _out, code = $server.run("cobbler buildiso --tempdir=#{tmp_dir} --iso #{iso_dir}/#{profile}.iso --distro=#{distro} --profile=#{profile} --exclude-dns")
+  raise 'error in cobbler buildiso, check the Cobbler logs' if code.nonzero?
+  # TODO Check for missing nameserver entry in isolinux.cfg
```

Comment:
We plan to do this TODO any time soon? Have we a card or something about it?
Otherwise this might just get lost in the code.

Reply:
Sorry, this is a leftover. Will fix it now.

Reply:
Fixed.